.github/workflows/pre-commit.yml

```diff
@@ -26,9 +26,22 @@ jobs:
         working-directory: ./
         shell: bash
 
-      - uses: actions/setup-python@v4
+      - name: Setup Python
+        uses: actions/setup-python@v4
+
+      - name: Create virtual environment
+        run: python3 -m venv venv
+
+      - name: Activate virtual environment and install pre-commit
+        run: |
+          source venv/bin/activate
+          python -m pip install pre-commit
+        shell: bash
+
       - name: yq - portable yaml processor
         uses: mikefarah/yq@v4.43.1
+        with:
+          cmd: ""
```

Comment:
is this needed?

Reply:
When I cloned down this repository it was complaining about this:

Missing required input `cmd`
A name for your step to display on GitHub.

Available expression contexts: github, inputs, vars, needs, strategy, matrix, secrets, steps, job, runner, env

Available expression functions: hashFiles

So I added that in and left it blank